lib/Performance/Probability.pm

```diff
@@ -0,0 +1,221 @@
+package Performance::Probability;
+
+use 5.010;
+use strict;
+use warnings;
+
+use Math::BivariateCDF;
+use Math::Gauss::XS;
+
+use Exporter;
+
+our @ISA = qw(Exporter);
+
+our @EXPORT_OK = qw(get_performance_probability);
+
+our $VERSION = '0.01_1';
+
+=head1 NAME
+
+Performance::Probability - The performance probability is a likelihood measure of a client reaching his/her current profit and loss. 
+
+=head1 VERSION
+
+0.01_1
+
+=head1 SYNOPSYS
+
+  use Performance::Probability;
```

Comment:
Please, add simple, human-understandable example, i.e. for 2 x 2 matrix of input data.